package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qrhunt",
-  "version": "1.0.5",
+  "version": "0.0.25",
```

Comment:
Don't commit the pre-release version.  You can selectively add files for commit in git.